man/git-summary.1

```diff
@@ -49,12 +49,14 @@ Outputs a repo summary:
 .nf
 $ git summary
 
-project  : express
-repo age : 10 months ago
-commits  : 1893
-active   : 93 days
-files    : 111
-authors  :
+project     : express
```

Comment:
Let's update the md file and generate the doc via https://github.com/tj/git-extras/tree/master/man#how-to-generate-documentation

Reply:
And also this: https://github.com/tj/git-extras/blob/master/Commands.md#git-summary

Reply:
:) oh I see. Only the MD file must be manually adapted. When modifiying the x.1 file I already wondered how people can modify this file with all the tags and making no mistake. Thx for pointing this out.

Reply:
Don't forget the  https://github.com/tj/git-extras/blob/master/Commands.md#git-summary

Reply:
@spacewander Done.